dj/api/nodes.py

```diff
@@ -515,79 +545,6 @@ def add_dimension_to_node(
     )
 
 
-@router.post("/nodes/{name}/table/", status_code=201)
-def add_table_to_node(
-    name: str,
-    data: CreateTable,
-    *,
-    session: Session = Depends(get_session),
-    query_service_client: QueryServiceClient = Depends(get_query_service_client),
-) -> JSONResponse:
-    """
-    Add a table to a node
-    """
-    node = get_node_by_name(session=session, name=name)
-    database = get_database_by_name(session=session, name=data.database_name)
-    catalog = get_catalog(session=session, name=data.catalog_name)
-    for existing_table in node.current.tables:
-        if (
-            existing_table.database == database
-            and existing_table.catalog == catalog
-            and existing_table.table == data.table
-        ):
-            raise DJException(
-                message=(
-                    f"Table {data.table} in database {database.name} in "
-                    f"catalog {catalog.name} already exists for node {name}"
-                ),
-                http_status_code=HTTPStatus.CONFLICT,
-            )
-
-    # When no columns are provided, attempt to find actual table columns
-    # if a query service is set
-    columns = [
-        Column(name=column.name, type=ColumnType(column.type))
-        for column in data.columns
-    ]
-    if not columns:
-        if not query_service_client:
```

Comment:
Not a blocker, but just flagging that we should add this again so that we can infer columns for loaded tables (in a follow-up PR)

Reply:
Another great catch! Let me move this over to the create node logic for source nodes since the table info is back on their definitions now.